pkg/sitewise/client/client.go

```diff
@@ -41,23 +42,139 @@ func NewSitewiseClientForRegion(region string) SitewiseClient {
 	}
 }
 
+type IndexedData struct {
+	Index int
+	Data  *iotsitewise.BatchGetAssetPropertyValueHistoryOutput
+}
```

Comment:
Very minor, but this type and its members might as well be lowercased since it's only of internal use.